agora.md

```diff
@@ -0,0 +1,288 @@
+
+After all of that, you don't own any Bitcoin yet. You receive an IOU and still need to transfer the funds into a wallet where you control the private keys. This experience is not ideal and goes against the Ethos of Bitcoin. 
+
+Additionally, your personal information is now stored on the exchange's server, making it vulnerable to potential breaches. Unfortunately, such incidents are not uncommon in the Bitcoin world. Companies like 'Mt Gox', 'Binance', and 'Ledger' have all experienced data breaches in the past.
+
+## So what's the alternative?
+Buying Bitcoin [noKYConly!](/nokyconly) 
+
+There are many ways to obtain Bitcoin without providing all of your details to a company that may or may not know how to secure data properly. I have guides on using peer-to-peer markets like [Bisq](/bisq), [HodlHodl](/hodlhodl), [Robosats](/robosats) and [Peach Bitcoin](/peach) but another noKYC alternative is AgoraDesk.
+
+<p align="center">
+<img src="/assets/img/NKYC2.png" class=responsive width="300" height="230" maxheight="300" />
+</p>
+
+## What is AgoraDesk?
+
+AgoraDesk is an Open Source and online peer-to-peer marketplace for trading cryptocurrencies, especially designed to address the issue of KYC in Bitcoin. AgoraDesk was created by the same team that developed LocalMonero, which is an equivalent to LocalBitcoins for Monero. 
+
+With a vision to uphold the principles of privacy and freedom, AgoraDesk does not require any personal information while offering a wide range of payment methods with good liquidity on the most common. AgoraDesk is available on the clearnet, Tor network, and mobile applications on iOS and Android. 
+
+## Before You Start 
+
+### Things you need to know
+
+#### Jargon Buster
+
+|----|-------|
+| [Arbitration Bond](https://learn.robosats.com/docs/robots/) | Arbitration bond is a certain amount of Bitcoin that a seller locks in the platform as a guarantee of their credibility and seriousness about a trade. |
+| [Trade Escrow](https://learn.robosats.com/docs/robots/) | Escrows are used by the seller as a method of holding the trade amount of Bitcoin. |
+| [Advertisements](https://learn.robosats.com/docs/robots/) | Sell offers on AgoraDesk. |
+| [Dispute Resolution](https://learn.robosats.com/docs/robots/) | AgoraDesk provides dispute resolution if there is a disagreement between the buyer and seller. They investigate and make a decision based on evidence provided by both parties. |
+| [Fees](https://learn.robosats.com/docs/robots/) | AgoraDesk users who create advertisements are charged a 1% arbitration protection fee for every completed trade. |
+
+<br/>
+
+### Things you need to have
+
+#### Bitcoin Wallet 
+
+When purchasing non-KYC Bitcoin on Agora, you must provide a receiving Bitcoin address. You can send sats directly to your cold storage or hot wallets such as [Samourai Wallet]() or [Sparrow Wallet]() to start mixing your freshly new acquired sats!
```

Comment:
Would be cool to include Envoy here!

Reply:
Sure, let's start including Envoy more often! 😎